webrender/src/device/gl.rs

```diff
@@ -2223,7 +2223,7 @@ impl Device {
                 pixel_type: gl::UNSIGNED_BYTE,
             },
             ImageFormat::R16 => FormatDesc {
-                internal: gl::RED as _,
+                internal: gl::R16 as _,
```

Comment:
let's fix the `internal` format of `R8` as well since we are at it (hint: should be `R8`)

Reply:
Good point, I'll push another change for that.